src/components/MyAccount/SettingsTab/PasswordChangeForm.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from "react"
+import {
+  Form,
+  FormField,
+  TextInput,
+  Button,
+} from "@nypl/design-system-react-components"
+import styles from "../../../../styles/components/MyAccount.module.scss"
+import { BASE_URL } from "../../../config/constants"
+import type { Patron } from "../../../types/myAccountTypes"
+
+const PasswordChangeForm = ({
+  patron,
+  updateModal,
+}: {
+  patron: Patron
+  updateModal: (errorMessage?: string) => void
+}) => {
+  const [formData, setFormData] = useState({
+    oldPassword: "",
+    newPassword: "",
+    confirmPassword: "",
+    passwordsMatch: true,
+  })
+
+  const validateForm = () => {
+    return (
+      formData.oldPassword !== "" &&
+      formData.newPassword !== "" &&
+      formData.confirmPassword !== "" &&
+      formData.passwordsMatch
+    )
+  }
+
+  const handleInputChange = (e) => {
+    const { id, value } = e.target
+    if (id === "confirmPassword") {
```

Comment:
This works well but you're missing something here. I'll run through the bug I found but this is where you should fix it.

- type in the old password/pin
- type in the new password/pin as 1234
- type in the confirmation password/pin as 12345
- they don't match. instead of updating the confirmation, what if a user goes back and updates the initial new password/pin field?
- type in the initial password/pin as 12345
- the error persists

Reply:
Is it okay that the confirm password shows up as incorrect while the new password is being entered originally?

Reply:
Yes, I think so.